niceman/resource/docker_container.py

```diff
@@ -178,13 +183,16 @@ def _execute_command(self, command, env=None, cwd=None):
         #    docker.errors.APIError - If the server returns an error.
         lgr.debug('Running command %r', command)
         execute = self.client.exec_create(container=self.container, cmd=command)
+        out = ''
         for i, line in enumerate(
                 self.client.exec_start(exec_id=execute['Id'],
                 stream=True)
         ):
             if line.startswith(b'rpc error'):
                 raise CommandError(cmd=command, msg="Docker error - %s" % line)
+            out += utils.to_unicode(line, "utf-8")
```

Comment:
hm, codecov reports this line to be not covered? could there be a basic test which executes some command with multiple lines of output?

Reply:
Added a multi line coverage.